app/src/main/java/com/example/wd_mvvm/models/NewsApiResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.wd_mvvm.models
+
+import com.example.wd_mvvm.models.article.Article
+
+data class NewsApiResponse (
+    var status: String,
+    var totalResults: Int,
```

Comment:
Take care to specify the serialized names in the future using @SerializedName for every field